### PR TITLE
Feature/XIVY-15683 upgrade backend and build it

### PR DESCRIPTION
**Pull request by weissreto**
Instead of using a prebuild backend (*.war files) build the backend on the fly for ivy-tracing-jaeger and ivy-monitor-prometheus samples
Upgrade to Grafana 11.4

### CODE REVIEW

File: ivy-monitoring-prometheus/backend/soap-service/.vscode/settings.json
Comment (ivy-lgi) [inline ivy-monitoring-prometheus/backend/soap-service/.vscode/settings.json:1]: needed?

File: ivy-monitoring-prometheus/backend/soap-service/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Country Service -> Generic WS Test project
+
+URLs look like:
```
Comment (ivy-lgi) [inline ivy-monitoring-prometheus/backend/soap-service/README.md:3]: URLs look like what? :sweat_smile:

File: ivy-monitoring-prometheus/backend/rest-service/src/main/java/ch/ivyteam/test/rest/service/PersonService.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.Response.Status;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Simple RESTful service. The REST interface is defined by the JAX-RS annotations on the methods and it's path.<br/>
+ * 
+ * @since 6.1.1
```
Comment (ivy-lgi) [inline ivy-monitoring-prometheus/backend/rest-service/src/main/java/ch/ivyteam/test/rest/service/PersonService.java:28]: needed?

File: ivy-monitoring-prometheus/backend/rest-service/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/target/
```
Comment (ivy-lgi) [inline ivy-monitoring-prometheus/backend/rest-service/.gitignore:1]: I suppose this is no longer needed as it is present in the root `.gitignore`?

File: ivy-monitoring-prometheus/backend/soap-service/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+/target/
```
Comment (ivy-lgi) [inline ivy-monitoring-prometheus/backend/soap-service/.gitignore:1]: Same as the other `.gitignore`.

File: ivy-tracing-jaeger/backend/rest-service/.gitignore [hunk 5]
```diff
@@ -0,0 +1 @@
+/target/
```
Comment (ivy-lgi) [inline ivy-tracing-jaeger/backend/rest-service/.gitignore:1]: Same as other `.gitignore`.

File: ivy-tracing-jaeger/backend/soap-service/.gitignore [hunk 6]
```diff
@@ -0,0 +1 @@
+/target/
```
Comment (ivy-lgi) [inline ivy-tracing-jaeger/backend/soap-service/.gitignore:1]: Same as other `.gitignore`.

File: ivy-tracing-jaeger/backend/soap-service/.vscode/settings.json
Comment (ivy-lgi) [inline ivy-tracing-jaeger/backend/soap-service/.vscode/settings.json:1]: needed?

File: ivy-tracing-jaeger/backend/soap-service/README.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# Country Service -> Generic WS Test project
+
+URLs look like:
```
Comment (ivy-lgi) [inline ivy-tracing-jaeger/backend/soap-service/README.md:3]: Same as before

**Review by ivy-lgi — APPROVED** [review]
LGTM
Some details I noticed.

File: ivy-monitoring-prometheus/backend/soap-service/README.md (hunk 1 above)
Comment (weissreto) [reply ivy-monitoring-prometheus/backend/soap-service/README.md]: Will remove whole file

File: ivy-monitoring-prometheus/backend/rest-service/src/main/java/ch/ivyteam/test/rest/service/PersonService.java (hunk 2 above)
Comment (weissreto) [reply ivy-monitoring-prometheus/backend/rest-service/src/main/java/ch/ivyteam/test/rest/service/PersonService.java]: No

**Pull request merged**